client/extensions/woocommerce/woocommerce-services/lib/initialize-carrier-accounts-state.js

```diff
@@ -1,41 +0,0 @@
-/** @format */
-
-/**
- * External dependencies
- */
-
-export default () => {
-	const requiredFields = [
-		'account_number',
-		'name',
-		'street1',
-		'city',
-		'state',
-		'country',
-		'postal_code',
-		'phone',
-		'email',
-		'title',
-		'website',
-	];
-
-	return {
-		UPS: {
```

Comment:
Hardcoding these values made it hard for the server to send new carrier types and have the app not break.